### ISSUE TITLE
Directional links

### ISSUE THREAD

**Issue by geekscrapy** [tchayen/markdown-links#61]
I'd really love the option of directional links - it'd help my notes a lot. Any chance of it being brought in? I appreciate that the README says explicitly that direction is _not_ given, is this is choice based on usage, or a note taking "schema"?

**Comment by nixsee** [comment]
I wouldn't want this. If implemented, please make it a setting to toggle on and off.

**Comment by geekscrapy** [comment]
Agree, this should definitely be optional, and probably default to off

**Comment by Am3ra** [comment]
@geekscrapy _why_ would this be helpful? Could you please expand on that? There may be more features hidden there!

**Comment by geekscrapy** [comment]
@Am3ra my notes generally centre around investigation of network attacks. They are free form notes (analysis writeups) which by tell a story of what happened on that host. At the end of analysis, I convert these markdown notes straight to a Word doc and submit to clients. The links between these notes come If an attacker moves between hosts. If the attacker goes from host A>B I can see the direction they were generally heading *straight from my textual notes!*. At the moment I can see where they were, but I don't get a sense of which host was the origin and the direction of travel throughout the network.
I want to say, I do have graph networks and databases to hand, but they don't allow me to directly write to a text file, which other people can instantly access without needing to know how to query, export etc. So, for me, it'd not only look better, it'd give me a great overview of what *happening* with my notes.

I can also see this being useful to create more mind map type note-taking.

**Comment by tchayen** [comment]
Links in Markdown are directed by nature, we just don't really care about that property.

@nixsee I agree, definitely something that would go to config.

@geekscrapy Thanks for reporting that and thanks so much for the detailed description. It makes me proud that a thing I initiated has so interesting use cases.

I don't have much time recently so won't be able to help but if anybody wants to check how creating such arrows looks in `d3-force` I'd be happy to help with any problems.

**Comment by srid** [comment]
Neuron uses `#[[..]]` and `[[..]]#` (the former is semantically equivalent to tagging as well) to denote direction and label of the link (the `#` here means folgezettel). See https://neuron.zettel.page/folgezettel
